site/en/r2/tutorials/load_data/csv.ipynb

```diff
@@ -183,26 +203,7 @@
         "id": "YOYKQKmMj3D6"
       },
       "source": [
-        "As you can see, the columns in the CSV are labeled. We need the list later on, so let's read it out of the file."
-      ]
-    },
-    {
-      "cell_type": "code",
-      "execution_count": 0,
-      "metadata": {
-        "colab": {},
-        "colab_type": "code",
-        "id": "v0sLG216MtwT"
-      },
-      "outputs": [],
-      "source": [
-        "# CSV columns in the input file.\n",
-        "with open(train_file_path, 'r') as f:\n",
-        "    names_row = f.readline()\n",
-        "\n",
-        "\n",
-        "CSV_COLUMNS = names_row.rstrip('\\n').split(',')\n",
-        "print(CSV_COLUMNS)"
+        "As you can see, the columns in the CSV are named. The dataset constructor will pick these names up automatically. If the file you are working with does not contain the column names in the first line, pass them in a list of strings to  the `column_names` argument in the `make_csv_dataset` function."
```

Comment:
Complete symbol name for autolink? `tf.data.experimental.make_csv_datase`

Reply:
I put the entire name in later. I didn't put it here because it's sort of an aside.